pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -298,3 +298,15 @@ func ValidateNodeAffinityArgs(args *config.NodeAffinityArgs) error {
 	}
 	return errors.Flatten(errors.NewAggregate(errs))
 }
+
+// ValidateNodeVolumeBindingArgs validates that VolumeBindingArgs are set correctly.
+func ValidateNodeVolumeBindingArgs(args *config.VolumeBindingArgs) error {
+	var path *field.Path
+	var err error
+
+	if args.BindTimeoutSeconds < 0 {
```

Comment:
Generally, loosening API validation is not allowed. But since this is not a kube-apiserver API, this is fine.

/kind bug

Please add a release note for the fix

Reply:
> Generally, loosening API validation is not allowed. But since this is not a kube-apiserver API, this is fine.

Good to know, thanks.

Reply:
> Please add a release note for the fix

Done